frontend/Pages/Index.elm

```diff
@@ -11,9 +11,14 @@ main =
     R.app app
 
 
-app : R.App Config Config Msg
+init : R.In -> Config -> ( Model, Cmd (R.Msg Msg) )
+init _ c =
+    ( { config = c }, Cmd.none )
+
+
+app : R.App Config Model Msg
 app =
-    R.App config R.init0 R.update0 R.sub0 document
+    R.App config init R.update0 R.sub0 document
```

Comment:
Lets remove ` R.update0 R.sub0` too.

Reply:
I have added the update and sub too. I have also switched their type definitions from type variable to the Model type used in the file.